### PR TITLE
Fixed arrows crossing over one another [#178285331]

### PR DESCRIPTION
**Pull request by dougmartin**
This changes the core jsPlumb node attachment point sorting algorithm so that edges don't crossover each other.

### CODE REVIEW

**Comment by dougmartin** [comment]
@knowuh - you'll want to disable whitespace diffs when reviewing.

File: src/vendor/jsPlumb.js [hunk 1]
```diff
@@ -8820,34 +8820,32 @@ var JSPLUMB_ZOOM = 1;
...
-                    else {
-                        r = a[0][0] > b[0][0];
+                    var p1 = a[0][0],
+                        p2 = b[0][0],
+                        result;
+                    if (flipValues) {
+                        // normalize the values on the negative y axis, which flips from -Pi to Pi when rotating counterclockwise
+                        p1 = p1 < 0 ? -Math.PI - p1 : Math.PI - p1,
+                        p2 = p2 < 0 ? -Math.PI - p2 : Math.PI - p2;
+                        // reversed on the left ride compared to the right side
```
Comment (kswenson) [inline src/vendor/jsPlumb.js:8832]:
```suggestion
                        // reversed on the left side compared to the right side
```

Comment (dougmartin) [reply src/vendor/jsPlumb.js:8832]: Hee hee.  I put this in to see if you would catch it @kswenson :)

**Review by knowuh — APPROVED** [review]
Looks good to me.

**Pull request merged**